app/src/main/java/com/ataulm/whatsnext/WhatsNextService.kt

```diff
@@ -48,17 +50,18 @@ internal class WhatsNextService(
             inWatchlist: Boolean,
             rating: FilmRating
     ) = withContext(Dispatchers.IO) {
-        letterboxdApi.updateFilmRelationship(
+        val apiFilmRelationshipUpdateResponse = letterboxdApi.updateFilmRelationship(
                 letterboxdId,
                 request = ApiFilmRelationshipUpdateRequest(
                         watched = watched,
-                        liked = liked ,
+                        liked = liked,
                         inWatchlist = inWatchlist,
                         rating = rating.toApiValue()
                 )
         )
+        // TODO: this is wasteful - the filmSummary won't have changed.
         val apiFilm = letterboxdApi.film(letterboxdId)
-        val apiFilmRelationship = letterboxdApi.filmRelationship(letterboxdId)
```

Comment:
Didn't need to do this call because the response we have already includes it. 

The `film` call is still wasteful though - we could just keep the cached data.